Common/include/gauss_jacobi_quadrature.hpp

```diff
@@ -0,0 +1,144 @@
+/*!
+ * \file gauss_jacobi_quadrature.hpp
+ * \brief Headers of the functions to compute the integration points of the
+          Gauss Jacobi quadrature rules.
+          The functions are in the <i>gauss_jacobi_quadrature.cpp</i> file.
+          All the functions in this class are based on the program JACOBI_RULE
+          of John Burkardt.
+ * \author E. van der Weide
+ * \version 6.1.0 "Falcon"
+ *
```

Comment:
@economon contacted Mr Burkhardt and his response was "please go ahead and use it". He didn't seem to care much about a citation. @economon, do you still have that email?

Reply:
@vdweide : yes, we should include the email exchange so that this usage is properly acknowledged. For example, see the READMEs in some of the externals/ directories that document usage of code.

I will forward you that again. Since this is not in the externals/ directory, can you please include wherever you see fit?

Reply:
@economon : I included your email conversation with John Burkardt in which he approves the usage of his code in both gauss_jacobi_quadrature.hpp and gauss_jacobi_quadrature.cpp. I assume this is OK now?